services/director-v2/src/simcore_service_director_v2/modules/comp_scheduler/_scheduler_base.py

```diff
@@ -800,11 +815,19 @@ async def _schedule_tasks_to_start(  # noqa: C901
         except (
             ComputationalBackendNotConnectedError,
             ComputationalSchedulerChangedError,
+            ClustersKeeperNotAvailableError,
         ):
             _logger.exception(
                 "Issue with computational backend. Tasks are set back "
                 "to WAITING_FOR_CLUSTER state until scheduler comes back!",
             )
+            await publish_project_log(
+                self.rabbitmq_client,
+                user_id,
+                project_id,
+                log="Unexpected error while scheduling computational tasks! TIP: contact osparc support if this does not resolve automatically.",
```

Comment:
if this reaches final user, use `user_message`  and run prompt to produce human-readable message

Reply:
well it is pretty human readable. and that only reaches the logs. but I will try to see what the AI might produce.